shell.js

```diff
@@ -23,7 +23,7 @@ exports.pwd = common.wrap('pwd', _pwd);
 
 //@include ./src/ls
 var _ls = require('./src/ls');
-exports.ls = common.wrap('ls', _ls, {idx: 1});
+exports.ls = common.wrap('ls', _ls, {noGlob: true});
```

Comment:
It's a real option.

Moving the all option into wrap would be really complex, and probobly not that useful. ls needs to do some weird stuff with common.expand, so I think we should just keep it in ls.